templates/gitops-template/gitops-promotion.yml.njk

```diff
@@ -38,7 +38,7 @@ jobs:
     # ubuntu-20.04 can also be used.
     runs-on: ubuntu-24.04
     container:
-      image: quay.io/redhat-appstudio/dance-bootstrap-app:rhtap-runner
+      image: quay.io/redhat-appstudio/rhtap-task-runner:latest
```

Comment:
Not sure which tag we should use. We can pin to a specific image, or if you want something like a `v1.2`, we'd need to edit the RPA config in the internal gitlab